buildpipeline/pipeline.json

```diff
@@ -20,7 +20,7 @@
           "Parameters": {
             "PB_DistroRid": "rhel.7.2-x64",
             "PB_DockerTag": "rhel7_prereqs_2",
-            "PB_AdditionalBuildArguments":"-PortableBuild=true",
+            "PB_AdditionalBuildArguments":"-PortableBuild=true -strip-symbols",
```

Comment:
Symbol stripping needs to be independently switchable for build from source, and I set the default to "don't strip" because stripping is fairly specific to our builds: https://github.com/dotnet/corefx/pull/15440#issuecomment-275202555.